supplement.csv

```diff
@@ -1,4 +1,5 @@
 Lemma,Sense,Word Class,Year,Quotation,Source,Note
+阿比西尼,/,proper noun,1857,?,Wylie『六合叢談』,Schmidt did not give quotation
```

Comment:
```suggestion
阿比西尼,/,proper noun,1857,?,A. Wylie『六合叢談』,
```
1) ? is quite enough to signal the lack of quotation
2) Include initials for easier reference later

The above comments also apply to other entries. For this particular entry, let’s drop it altogether since neither the Mandarin nor the English word is current in present-day prose.

Cf. README.md

> The intention is to focus on words and forms which are still current in everyday usage, although antedatings of obsolete forms are welcome.

Reply:
The ‘antedatings of obsolete forms’ part refers to findings made in published research. We don’t have time for that in this placement.

Reply:
Understood. Thank you!